subsys/caf/modules/ble_state_pm.c

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/types.h>
+
+#include <caf/events/ble_common_event.h>
+
+#define MODULE ble_state_pm
+#include <caf/events/module_state_event.h>
+#include <caf/events/power_manager_event.h>
+#include <caf/events/keep_alive_event.h>
+
+static unsigned int connection_count;
```

Comment:
move to tightest scope (inside function - line 21)

Reply:
ok